### PR TITLE
Added Profile Page

### LINKED ISSUE #2006: Adding My Profile Page

### PR DESCRIPTION
**Pull request by Saipradyumnagoud**
## Fixes Issue
Closes #2006 

## Screenshots

image

## Summary by CodeRabbit

- **New Features**
  - Introduced a new profile page for users to manage their profile information.
  - Updated navigation in the Aside component to direct users to the new profile page.

- **Improvements**
  - Enhanced user experience with an organized interface for editing profile details and managing coupon visibility.

### CODE REVIEW

**Comment by Saipradyumnagoud** [comment]
@codervivek5 Can you please review the PR soon and let me know for any more changes to be required

File: src/User/pages/Dashboard/ProfilePage.jsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
+import React, { useState } from 'react';
+import Aside from '../../components/Aside/Aside';
+
+const ProfilePage = () => {
+  const [formData, setFormData] = useState({
+    name: '',
+    email: '',
+    mobile: '',
+    password: '',
+    profilePicture: null, // Add a state to store the profile picture
```
Comment (codervivek5) [inline src/User/pages/Dashboard/ProfilePage.jsx]: add a sample avatar till then they do not add any image

**Review by codervivek5 — CHANGES_REQUESTED** [review]

File: src/User/pages/Dashboard/ProfilePage.jsx (hunk 1 above)
Comment (Saipradyumnagoud) [reply src/User/pages/Dashboard/ProfilePage.jsx]: Yes @codervivek5 i will add

**Comment by Saipradyumnagoud** [comment]
@codervivek5 I have updated a sample avatar

image

**Comment by codervivek5** [comment]
Thankyou for your contribution :)

**Pull request merged**